### PR TITLE
add grounding citations

### PR DESCRIPTION
**Pull request by rkaminsk**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -8631,6 +8631,12 @@ @incollection{falepe12a
   pages = {247-264}
 }
 
+@misc{falepepf01a,
+  title = {Efficient Instantiation of Disjunctive Databases},
+  author = {W. Faber and N. Leone and S. Perri and G. Pfeifer},
+  year = {2001}
+}
+
```
Comment (tortinator) [inline krr.bib:8639]: What about citing the TR at https://www.dbai.tuwien.ac.at/research/report/dbai-tr-2001-44.pdf

**Review by tortinator — APPROVED** [review]
Feel free to merge at your convenience - just check the TR of falepepf

File: krr.bib (hunk 1 above)
Comment (rkaminsk) [reply krr.bib:8639]: Yes, that's the one. How to best cite it?

Comment (rkaminsk) [reply krr.bib:8639]:
```bib
@techreport{falepepf01a,
  title = {Efficient Instantiation of Disjunctive Databases},
  author = {W. Faber and N. Leone and S. Perri and G. Pfeifer},
  year = {2001},
  number = {DBAI-TR-2001-44},
  institution = {Technische Universit{\"a}t Wien},
  type = {Technical Report}
}
```

**Pull request merged**